vscode/src/commands/context/shell.ts

```diff
@@ -35,11 +35,12 @@ export async function getContextFileFromShell(command: string): Promise<ContextI
         const cwd = vscode.workspace.workspaceFolders?.[0]?.uri?.fsPath
         const filteredCommand = command.replaceAll(/(\s~\/)/g, ` ${homeDir}${path.sep}`)
 
-        if (filter(commandsNotAllowed, cmd => filteredCommand.startsWith(cmd)).length > 0) {
-            throw new Error('Cody cannot execute this command')
-        }
-
         try {
+            if (filter(commandsNotAllowed, cmd => filteredCommand.startsWith(cmd)).length > 0) {
```

Comment:
```suggestion
            if (commandsNotAllowed.some(cmd => filteredCommand.startsWith(cmd)) {
```

I don't know what I was thinking 😂